integration-tests/devtools/src/test/java/io/quarkus/devtools/codestarts/QuarkusCodestartRunSingleIT.java

```diff
@@ -0,0 +1,89 @@
+package io.quarkus.devtools.codestarts;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+
+import com.google.common.collect.ImmutableList;
+
+import io.quarkus.devtools.PlatformAwareTestBase;
+import io.quarkus.devtools.ProjectTestUtil;
+import io.quarkus.devtools.codestarts.QuarkusCodestartData.DataKey;
+import io.quarkus.devtools.messagewriter.MessageWriter;
+import io.quarkus.devtools.project.BuildTool;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class QuarkusCodestartRunSingleIT extends PlatformAwareTestBase {
```

Comment:
You can remove that class, your codestart is already automatically tested by the  `QuarkusCodestartRunIT`. I am currently creating a tool to easily generate a codestart in development which will remove the need of using the tests for this..